Userland/Libraries/LibWeb/Page/EventHandler.cpp

```diff
@@ -183,10 +183,15 @@ bool EventHandler::handle_mousedown(const Gfx::IntPoint& position, unsigned butt
     }
 
     NonnullRefPtr document = *m_frame.active_document();
+    auto result = layout_root()->hit_test(position, Layout::HitTestType::Exact);
+    auto pointer_events = result.layout_node->computed_values().pointer_events();
+    if (pointer_events == CSS::PointerEvents::None) {
+        return false;
+    }
```

Comment:
```suggestion
    if (pointer_events == CSS::PointerEvents::None)
        return false;
```

no need for curly braces if the body is only one line